blocks/cards/cards.js

```diff
@@ -18,14 +20,21 @@ export async function fetchHtml(path) {
   return text;
 }
 
-export function renderCard(post) {
+async function extractTopicBasedOnLocale(topic) {
+  if (topic === undefined) return topic;
+  const apiResponse = await fetchAPI(`${TAGS_QUERY_INDEX}?sheet=topic`);
+  const tagTopic = apiResponse.data.find((tag) => tag.identifier === topic);
+  return tagTopic[getLocale()] || tagTopic['en-US'] || tagTopic.identifier || topic;
+}
+
+export async function renderCard(post) {
   return (
     div({ class: 'card' },
       div({ class: 'card-thumbnail' },
         a({ href: post.path },
           createOptimizedPicture(post.image, post.header),
         ),
-        post.topic ? div({ class: 'topic-tag' }, div(post.topic)) : '',
+        post.topic ? div({ class: 'topic-tag' }, div(await extractTopicBasedOnLocale(post.topic))) : '',
```

Comment:
Let's make this function a little bit shorter, like `localisedTopic(post.topic)`